site/themes/contour/layouts/shortcodes/notice.html

```diff
@@ -0,0 +1,21 @@
+<!--
+https://github.com/martignoni/hugo-notice
```

Comment:
this link seems to be broken?

Reply:
`site/themes/contour/layouts/shortcodes/notice.html` is the location of where the notice.html file should be installed, it isn't a link.